client/client.go

```diff
@@ -345,24 +345,25 @@ type (
 	TaskQueueVersionSelection = internal.TaskQueueVersionSelection
 
 	// TaskQueueDescription is the response to [Client.DescribeTaskQueueEnhanced].
-	// WARNING: Worker versioning is currently experimental.
```

Comment:
I assume this is being removed because this stuff is now considered GA? Isn't this all part of worker versioning technically the stats are per build ID, or is this RPC and the way it comes back considered GA?

Reply:
That's something I wanted to discuss actually. The versioning stuff are not GA for sure. I'd like to be able to say the Enhanced DecribeTaskQueue API parts that don't have to do with versioning is GA. It's a little tricky because of the `versions_info` map.

What do you guys think of a change like below to the API? It removes the empty string ugliness, also it allows us to keep the build_id related parts experimental and unversioned part GA.
![image](https://github.com/user-attachments/assets/b35b856b-e266-4437-a746-2a404d338074)
cc @cretz @Quinn-With-Two-Ns

Reply:
considering to rename `TaskQueueVersionInfo` to just `TaskQueueInfo` as well. That would fit better with unversioned.

Reply:
That API change works for me, but the empty string does too. I have no strong opinion, though I could understand not wanting to go through a cycle of API changes just to be able to split up the "unstable" comment.